src/Pages/sangkuoh/Main/Components/Feeds/MainLike/Like.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import MainHeartLike from './MainHeartLike';
+import MainNormalLike from './MainNormalLike';
+
+class Like extends Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      like: false
+    }
+  }
+
+  changeMainLike = () => {
+    this.setState({like : !this.state.like});
+  }
+
+  render() {
+    return (
+      <div>
+        {this.state.like ? <MainHeartLike onClick={this.changeMainLike}/> : <MainNormalLike onClick={this.changeMainLike}/>}
+      </div>
```

Comment:
```suggestion
      <div>
        {this.state.like 
            ? <MainHeartLike onClick={this.changeMainLike}/> 
            : <MainNormalLike onClick={this.changeMainLike}/>
        }
      </div>
```

-  가독성 위해서 줄바꿈 잘해주세요 :)

Reply:
수정하였습니다.⚡️